media/js/firefox/sticky-promo.js

```diff
@@ -0,0 +1,60 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    // Check if promo exists on the page or if smaller than tablet.
+    var matchMediaDesktop = window.matchMedia('(min-width: 768px)').matches;
+
+    if (!document.querySelector('.mzp-c-sticky-promo') || !matchMediaDesktop) {
```

Comment:
I see two calls to `document.querySelector('.mzp-c-sticky-promo')` in the script (the other in `bindEvents`), so suggest using a variable for reference to avoid needing to query the DOM twice? e.g.

```javascript
var promo = document.querySelector('.mzp-c-sticky-promo');

if (!promo || !matchMediaDesktop) {

}
````